### PR TITLE
Jest Tests

### PR DESCRIPTION
**Pull request by akroon3r**
This PR contains the commits that hold the new bookings jest tests for creating, editing, deleting and returning an exam.

### CODE REVIEW

File: frontend/src/exams/add-exam-modal.vue [hunk 1]
```diff
@@ -285,11 +290,13 @@
         let { setup } = this.addExamModal
         this.unSubmitted = false
         this.submitMsg = ''
+        console.log("SETUP", setup)
```
Comment (sjrumsby) [inline frontend/src/exams/add-exam-modal.vue]: Not gonna fail this PR for this, but can you remove this console log in your next jest push?

File: frontend/src/store/index.js [hunk 2]
```diff
@@ -1675,6 +1685,7 @@ export const store = new Vuex.Store({
     },
 
     postITAGroupExam(context) {
+      console.log("First line")
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: Same goes for this log line

File: frontend/src/store/index.js [hunk 3]
```diff
@@ -1683,6 +1694,7 @@ export const store = new Vuex.Store({
       let time = new moment(responses.exam_time).format('HH:mm:ss')
       let datetime = date+'T'+time
       let start
+      console.log("Datetime", datetime)
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: And this guy

File: frontend/src/exams/add-exam-modal.vue (hunk 1 above)
Comment (akroon3r) [reply frontend/src/exams/add-exam-modal.vue]: fixed.

File: frontend/src/store/index.js (hunk 2 above)
Comment (akroon3r) [reply frontend/src/store/index.js]: fixed

File: frontend/src/store/index.js (hunk 3 above)
Comment (akroon3r) [reply frontend/src/store/index.js]: fixed

**Pull request merged**